lib.php

```diff
@@ -159,17 +159,15 @@ function kanban_inplace_editable($itemtype, $itemid, $newvalue) {
 
     \mod_kanban\helper::check_permissions_for_user_or_group($boardmanager->get_board(), $context, $boardmanager->get_cminfo());
 
-    $newtitle = clean_param($newvalue, PARAM_TEXT);
-
     if ($itemtype == 'card') {
-        $boardmanager->update_card($itemid, ['title' => $newtitle]);
+        $boardmanager->update_card($itemid, ['title' => $newvalue]);
     }
 
     if ($itemtype == 'column') {
-        $boardmanager->update_column($itemid, ['title' => $newtitle]);
+        $boardmanager->update_column($itemid, ['title' => $newvalue]);
     }
 
-    return new \core\output\inplace_editable('mod_kanban', $itemtype, $itemid, true, $newtitle, $newtitle, null, '');
+    return new \core\output\inplace_editable('mod_kanban', $itemtype, $itemid, true, s($newvalue), $newvalue, null, '');
```

Comment:
the PHP doc of inplace_editable suggest to use format_string here instead of s(...). Any reason not to do this?

Due to my manual testing this would also mean that we do not need the triple mustache brackets in the mustache file, right?

Reply:
format_string() would also use filters - that could lead to unexpected results.

Reply:
yes, I know. Is this not intended?

Reply:
I don't think filtering the title input is necessary - could be disturbing in some cases

Reply:
alright